schemas/url.yml

```diff
@@ -68,6 +68,24 @@
         simply taking the last two labels will not work well for TLDs such as "co.uk".
       example: google.com
 
+    - name: top_level_domain
+      level: extended
+      type: keyword
+      short: The top level domain is the last part of the domain (com, net, org).
+      description: >
+        The top level domain (TLD) also known as the domain suffix is the last part of the domain name.
+        For example, the top level domain for google.com is "com".
+
+        The following groups of top level domain are maintained by the Internet Assigned Numbers Authority (IANA).
```

Comment:
I think this sentence and the subsequent list should be replaced by a mention of Mozilla's public suffix list. You can take inspiration from the `registered_domain` definition for wording.

Of course both approaches can be used to figure out the eTLD, but Mozilla's list is the more direct method. One file, and you're able to determine the eTLD. Pointing users to a slew of lists like this will discourage them from even trying, IMO.